apps/main-app/src/routes.js

```diff
@@ -54,6 +54,15 @@ export default (
         }}
       />
 
+      <Route
+        exact
+        path='/jbrowse'
+        render={() => {
+          window.location.href = '/jbrowse/?data=data%2FHomo%20sapiens';
+          return null;
+        }}
+      />
+
```

Comment:
Ahh... didn't know there was a "route" for blue genes. This makes sense... @scottcain